docs/dev/troubleshooting/changelog.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+## System update v1.3 (8th Feb 2023)
+
+This update included multiple changes in the system to prepare it for our “Fair Onboarding Alpha” milestone. These updates include:
+
+- a fee mechanism revamp that replaces `ergs` with `gas` to make it easier to understand (after all, we’re all part of the Ethereum ecosystem).
+- updates in the Account Abstraction support
+- changes in L1 and L2 system contract interfaces
+- breaking changes in our SDKs where methods and properties that contained `ergs` are renamed `gas`
+- and some API error messages now follow the Geth format (more updates coming soon).
+
+**These udpates were followed by a system regenesys.**
+
+### How to update your project
+
+To update your project, follow these steps:
+
+- Update all your project dependencies, while continuing to use `ethers v5.7.x` (as `ethers v6.x` introduces additional breaking changes).
+- Delete the `artifacts-zk` and `cache-zk` folders.
+- If you’re using the local-setup to run unit tests, update the docker images with `docker-compose pull` and restart the local setup state by running the `./clear.sh` script.
```

Comment:
@uF4No did we update + test the local node?

Reply:
I was told the docker images were going to be updated

Reply:
confirmed with @Deniallugo that they'll released the docker images later today